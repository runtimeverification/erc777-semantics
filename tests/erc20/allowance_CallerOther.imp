test.preamble();

// Testing allowance(Owner,Spender) when Owner = caller but Spender different.

test.allowance(caller, 4);
