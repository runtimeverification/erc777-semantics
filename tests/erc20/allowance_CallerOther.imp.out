Allowance of 7 for 4 is 0
